ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -1461,6 +1461,9 @@ VOCM:
   guid: "d1d343aa-9b3a-41aa-99fe-850c2ba355a8"
   description: "Volatile organic compound monitoring."
   is_abstract: true
+  opt_uses:
```

Comment:
Please do not add these here. Instead use `VOC` and `PMM`

Reply:
updated